pledgeindia/user/models.py

```diff
@@ -51,6 +51,8 @@ class User(AbstractBaseUser, PermissionsMixin, TimeStampedModel):
     last_name = models.CharField(_('last name'),  max_length=30, blank=True)
     date_joined = models.DateField(_('date joined'), auto_now=True)
     is_active = models.BooleanField(_('active'), default=True)
+    is_staff = models.BooleanField(_('staff status'), default=False,
```

Comment:
This is already present in the master branch. Did you sync your fork with origin master ?
https://help.github.com/articles/syncing-a-fork/